app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
> I still think, that for comments you should be able to mention the post-author and the last few comment-authors without typing anything (only the @), so without q.

It's not possible the way the mention class is written right now. This mean we should change the logic and add an exception case and I'm not for at all. Plus, on a personnal point of view, I find this counter intuitive to be able to mention people without a `@` on comments and not in publisher.

> I would prefer a solution with the available data from the frontend (because everything should already be there)

Same problem: no data is available on the front-end right now. There was, not long ago, but #6951 changed that behaviour. This mean refactor (again) the search system.